website/content/docs/integration-guide.md

```diff
@@ -0,0 +1,46 @@
++++
+toc = true
+weight = 3
+draft = false
+title = "The integration guide"
+date = "2018-02-09T10:15:00+01:00"
+description = "Do you want to learn how to extend or integrate with bloop?"
+bref = "Learn how to integrate your own tooling with bloop."
++++
+
+## An overview of integrations
+
+There are two main ways you can extend bloop or integrate it with external
```

Comment:
Generally, I think it's good to start pages with "This page is intended for XXX people". It helps a regular user decide whether they can skip the contents or not.